src/components/inject.js

```diff
@@ -294,31 +294,32 @@ Please be sure the component that calls createSource or createToken is within an
 
       const elementOrDataResult = this.parseElementOrData(elementOrData);
 
-      // Second argument is Element; handleCardPayment with element
+      // Second argument is Element; handle with element
       if (elementOrDataResult.type === 'element') {
         const {element} = elementOrDataResult;
         if (maybeData) {
-          return stripe.handleCardPayment(clientSecret, element, maybeData);
+          return stripe[method](clientSecret, element, maybeData);
```

Comment:
Not the biggest fan of dynamically calling methods like this, but I guess duplication/more nested if-else blocks isn't great either. 🤷‍♂

Reply:
== but the underlying logic is was so identical that it felt really silly to duplicate everything. It is typed at least.